kshebeko/lesson4/src/com/company/kshebeko_class/Bouquet.java

```diff
@@ -0,0 +1,47 @@
+package com.company.kshebeko_class;
+
+import java.util.ArrayList;
+
+public class Bouquet {
+    public Accessories accessories;
+    public String freshFlower;
+    public int price;
+    public String name;
+
+    public Bouquet (String name, int price){ //Конструктор
```

Comment:
this constructor will never be used, because price of bouquet will be calculated inside of class and not passed from outside